mongo/post_startup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Update users if database has been already created.
+# Users creation on init is left to `init_02_create_user.sh`
+BASE_DIR="$(readlink -f $(dirname "$BASH_SOURCE"))"
+MONGO_CMD=( mongo --host 127.0.0.1 --port 27017 --quiet -u "$KOBO_MONGO_USERNAME" -p "$KOBO_MONGO_PASSWORD")
+MONGO_ADMIN_DATABASE=admin
+COLLECTION=instances
+
+while [ "$((echo > /dev/tcp/127.0.0.1/27017) >/dev/null 2>&1 && echo "1" || echo "0")" == "0" ]; do
```

Comment:
how about this?
```bash
until (echo > /dev/tcp/127.0.0.1/27017) 2> /dev/null; do
…
```
![image](https://user-images.githubusercontent.com/2085013/124012075-b6d6cf00-d9ae-11eb-9c33-93a80eea43f6.png)

Reply:
also is 27017 always going to be the port for mongo?

Reply:
> Until ... 

Way better 


> also is 27017 always going to be the port for mongo?

Inside the container, yes! We only change the exposed port, but it's always mapped to 27017 within the container.

Reply:
Great, I forgot that :+1: